src/frontend/components/Felleskomponenter/Barnehagemodal/barnehageplassSpråkUtils.ts

```diff
@@ -1,12 +1,45 @@
+import { LocaleRecordBlock, LocaleRecordString } from '../../../typer/common';
+import { IBarnehageplassTekstinnhold } from '../../../typer/sanity/modaler/barnehageplass';
 import { EBarnehageplassPeriodeBeskrivelse } from './barnehageplassTyper';
 
-export const beskrivelseSpråkId = (beskrivelse: EBarnehageplassPeriodeBeskrivelse | ''): string => {
+export const hentBarnehageplassBeskrivelse = (
+    beskrivelse: EBarnehageplassPeriodeBeskrivelse | '',
+    tekster: IBarnehageplassTekstinnhold
+): LocaleRecordString => {
     switch (beskrivelse) {
         case EBarnehageplassPeriodeBeskrivelse.HATT_BARNEHAGEPLASS_TIDLIGERE:
-            return 'todo.ombarnet.barnehageplass.periode.tidligere';
+            return tekster.valgalternativBarnehageplassTidligere;
         case EBarnehageplassPeriodeBeskrivelse.TILDELT_BARNEHAGEPLASS_I_FREMTIDEN:
-            return 'todo.ombarnet.barnehageplass.periode.fremtiden';
+            return tekster.valgalternativBarnehageplassIFremtid;
```

Comment:
For lesbarhetens skyld synes jeg det hadde vært litt nice om også casen som nå går til default hadde vært skrevet opp 😊